### PR TITLE
Adds advice to check your CAPA problems after re-run

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3143](https://openedx.atlassian.net/browse/DOC-3143)

Recent a11y changes to CAPA markdown/up and to the options that learners have while working on problems prompted these additions to the re-run section.
In addition, based on recent feedback from the Open source community, I have un-commented the instructions for how to re-run. 
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [x] Doc team review (copy edit): @edx/doc
- [x] Product review: @sstack22 
- [ ] Partner support: 
- [ ] PM review: 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/course_components/create_problem.rst [hunk 1]
```diff
@@ -78,7 +78,7 @@ The editing interface that opens depends on the type of problem you choose.
 
 * For advanced problem types (with the exception of :ref:`peer assessment<Open
   Response Assessments 2>`), the :ref:`advanced editor<Advanced Editor>` opens.
-  In this editor you use OLX (open learning XML) tags, elements, and attributes
+  In this editor you use OLX (open learning XML) elements and attributes
```
Comment (srpearce) [inline en_us/shared/course_components/create_problem.rst]: Nit: definition before acronym ("open learning XML (OLX)")

File: en_us/shared/rerun_course/course_rerun.rst [hunk 2]
```diff
@@ -185,15 +196,18 @@ course for release.
...
+  :ref:`Markdown-style formatting <Simple Editor>` or :ref:`OLX markup
+  <Advanced Editor>` of any problems created before September 2016.
+  For more information about the updates that you can make to improve the
+  accessibility of these problem types, see the `Release Notes
+  <http://edx.readthedocs.io/projects/edx-release-notes/en/latest/studio_index.html#updates-to-capa-problem-types>`_.
+
+* If your course includes instructions for learners, verify that the
+  instructions reflect the current user interface of the LMS. For example, you
+  might want to revise a description of the problem **Check** button, which was
+  accurate before October 2016, to reflect its new label, **Submit**.
```
Comment (srpearce) [inline en_us/shared/rerun_course/course_rerun.rst]: Do we want to mention a change that isn't scheduled until next month, given that sometimes changes don't happen when they're originally scheduled?

File: en_us/shared/rerun_course/course_rerun.rst [hunk 3]
```diff
@@ -163,6 +161,19 @@ course for release.
 * Delete or edit posts on the **Course Updates** page in Studio. See :ref:`Add
   a Course Update`.
 
+* For a course that includes :ref:`learner cohorts<Enabling and Configuring
+  Cohorts>`, set up the cohorts and select a strategy for assigning learners to
+  the cohorts.
+
+* For a course that includes drag and drop problems, replace any problems
+  created prior to April 2016, with the newer drag and drop problem component,
```
Comment (srpearce) [inline en_us/shared/rerun_course/course_rerun.rst]: Strike comma after 2016

**Review by srpearce — APPROVED** [review]

File: en_us/shared/rerun_course/course_rerun.rst (hunk 2 above)
Comment (lamagnifica) [reply en_us/shared/rerun_course/course_rerun.rst]: agreed. wanted to move to doc complete but hold off on the merge until we see those last CAPA changes go

**Comment by sstack22** [comment]
👍

**Pull request merged**